common/build/eslint-config-fluid/minimal.js

```diff
@@ -346,6 +346,7 @@ module.exports = {
             rules: {
                 "@typescript-eslint/no-invalid-this": "off",
                 "@typescript-eslint/unbound-method": "off", // This rule has false positives in many of our test projects.
+                "import/no-nodejs-modules": "off", // Node libraries are OK for test files.
```

Comment:
With this off for tests, we should consider making it an error instead of a warning (on line 327).

Reply:
note that making it an error instead of a warning would be a breaking change, so might require some extra stuff (breaking change notes, fixing things when upgrading etc).

Reply:
Yeah, let's skip this for now.